apps/passport-client/public/service-worker.js

```diff
@@ -0,0 +1,11 @@
+self.addEventListener('install', (event) => {
+  console.log('Service worker installed');
+});
+
+self.addEventListener('activate', (event) => {
+  console.log('Service worker activated');
+});
```

Comment:
fine for now, but if we want any actual logic in the service worker we should write it in Typescript.

we can add service worker compilation to the `esbuild` build script